docs/providers/aws/events/http-api.md

```diff
@@ -392,6 +392,18 @@ provider:
     payload: '1.0'
 ```
 
+Also, the payload version could be configured in function level as follows:
```

Comment:
Let's maybe reword it as

The payload version can also be specified at the function level with `httpApi.payload` property and it will take precedence over the payload version configured at the provider level. It can be configured as: